.github/workflows/code_coverage.yml

```diff
@@ -0,0 +1,315 @@
+# Contains jobs corresponding to code coverage.
+
+name: Code Coverage
+
+# Controls when the action will run. Triggers the workflow on pull request
+# events or push events in the develop branch.
+on:
+  pull_request:
+  push:
+    branches:
+      # Push events on develop branch
+      - develop
+
+jobs:
+#  check_unit_tests_completed:
+#    name: Check unit test completed
+#    runs-on: ubuntu-latest
+#    steps:
+#      - name: Wait for unit tests to checks
+#        uses: ArcticLampyrid/action-wait-for-workflow@v1.2.0
+#        with:
+#          workflow: unit_tests.yml
+#          sha: auto
```

Comment:
Is this disabled for temporary testing?

Reply:
Yeah, it was hard to verify while waiting for unit tests to pass, will make sure to revert it once all coverage workflows are verified.

I did confirm that it only gets triggered after the unit tests are successfully completed.